src/components/Header/Header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import classes from './Header.css';
+
+const header = () => {
+  return (
+    <header className={classes.header}>
+      <div className={classes.wrap}>
+        <span className={classes.logo}>netflixroulette</span>
+        <h2>Find your move</h2>
+        <input type="text" value="Quentin Tarantino" />
```

Comment:
Input should be editable. See the one-way-data binding pattern [here](https://reactjs.org/docs/lifting-state-up.html#adding-a-second-input)